Makefile

```diff
@@ -121,6 +121,7 @@ ifeq ($(OPERATING_SYSTEM),Linux)
 	sudo apt install -y clang-tidy
 	sudo apt install -y ninja-build
 	sudo apt install -y valgrind
+	sudo apt install -y unzip
```

Comment:
We are fairly opinionated on using `apt` here. This should be documented in the readme as a prereq